README.md

```diff
@@ -694,6 +695,30 @@ For pages, it is almost as straightforward. Click on "Pages" in the admin bar to
 
 As with pieces, change "Trash" to "No." The "Trash" field will be located right after the "Published" field. When you click save, the page will be live in this locale.
 
+## Removing workflow from a project
```

Comment:
We should include some additional steps. The main thing I see is the warning to remove from app.js before starting the site back up. We should have a warning that down-time is probably needed. So what are the steps on a production site?

1. Plan for down time.
2. Run the task on the server
3. Update your project to disable workflow (and probably any related code) and deploy.
4. Start the site back up.